client/package.json

```diff
@@ -30,6 +30,7 @@
     "react-redux": "^7.2.1",
     "react-router": "^5.2.0",
     "react-scripts": "^3.4.3",
+    "react-social-icons": "^5.4.0",
```

Comment:
Same thing as the other comment, I don't think you need to commit the changes to package or package-lock. I could be wrong though